### DISCUSSION TITLE
Where to send trash reports

### DISCUSSION THREAD

**Discussion by Ingood5** [d4rken-org/sdmaid-se#533] [Q&A] [answered]
На данный момент отчёты о найденном мусоре ещё можно отправлять через SD Maid 1?

**Comment by d4rken** [comment, marked as answer]
I don't have a clutter report tool for SD Maid 2 yet.
You can use the tool from SD Maid 1 if it is easier and I will add them to SD Maid 2.

If the format doesn't change I might do another update with the definitions for SD Maid 1 too, but the priority is SD Maid 2.
